src/test/java/Homework20.java

```diff
@@ -4,48 +4,64 @@
         //WebElement addPlayList = driver.findElement(By.cssSelector("#playlists i[role='button']"));
-        addPlayList.click();
+//        actions.moveToElement(homePage.getAddPlaylist()).perform();
+//        addPlayList.click();
         //Thread.sleep(3000);
 
-        WebElement createPlayListBtn = wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("li[data-testid='playlist-context-menu-create-simple']")));
+//        WebElement createPlayListBtn = wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("li[data-testid='playlist-context-menu-create-simple']")));
         //WebElement createPlayListBtn = driver.findElement(By.cssSelector("li[data-testid='playlist-context-menu-create-simple']"));
-        createPlayListBtn.click();
+//        createPlayListBtn.click();
 
-        WebElement playListNameInput = wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("[name='create-simple-playlist-form']>input")));
+//        WebElement playListNameInput = wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("[name='create-simple-playlist-form']>input")));
         //WebElement playListNameInput = driver.findElement(By.cssSelector("[name='create-simple-playlist-form']>input"));
-        playListNameInput.sendKeys(playListName);
-        playListNameInput.sendKeys(Keys.ENTER);
+//        playListNameInput.sendKeys(playListName);
+//        playListNameInput.sendKeys(Keys.ENTER);
         //Thread.sleep(3000);
 
         //WebElement choosePlayList = wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath("//section[@id='playlists']//li//a[text()='Playlist 78']")));
-        WebElement choosePlayList = driver.findElement(By.xpath("//section[@id='playlists']//li//a[text()='Playlist 78']"));
-        choosePlayList.click();
+//        WebElement choosePlayList = wait.until(ExpectedConditions.visibilityOfElementLocated
+//                (By.xpath(String.format("//section[@id='playlists']//li//a[text()='%s']", playListName))));
+//        choosePlayList.click();
         //Thread.sleep(3000);
 
         //Delete Play list
-        WebElement deleteBtn = wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath("//button[contains(@class, 'del btn-delete-playlist')]")));
+//        WebElement deleteBtn = wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath("//button[contains(@class, 'del btn-delete-playlist')]")));
         //WebElement deleteBtn = driver.findElement(By.xpath("//button[contains(@class, 'del btn-delete-playlist')]"));
-        deleteBtn.click();
+//        deleteBtn.click();
         //Thread.sleep(3000);
         //wait.until(ExpectedConditions.invisibilityOfElementLocated(By.xpath("//section[@id='playlists']//li//a[text()='Playlist 78']")));
-        Assert.assertTrue(wait.until(ExpectedConditions.invisibilityOfElementLocated(By.xpath("//section[@id='playlists']//li//a[text()='Playlist 78']"))));
```

Comment:
I recommend not using code comments: for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.